llvm/lib/Target/PowerPC/PPCInstr64Bit.td

```diff
@@ -218,7 +218,7 @@ let isCall = 1, PPC970_Unit = 7, isCodeGenOnly = 1,
                               (ins (memrix $D, $RA):$src),
                               "bctrl\n\tld 2, $src", IIC_BrB,
                               [(PPCbctrl_load_toc iaddrX4:$src)]>,
-    Requires<[In64BitMode]>;
+    Requires<[In64BitMode]>, MemOp;
```

Comment:
This could be moved into `XLForm_2_ext_and_DSForm_1`. The only uses are here and the next instruction.